src/components/Shared/MarbleSearchFacets/index.js

```diff
@@ -11,7 +11,7 @@ const MarbleSearchFacets = () => {
         field='centuryTag.keyword'
         label='Time Period'
         operator='OR'
-        sort='a-z'
```

Comment:
Not a blocker.  I am pretty sure that I did this for Abby for a very specific reason.  Though it was years ago.   So I would say maybe get this on test and show Mikala and see what she says.

Reply:
It's set up in such a way that the dates appear in chronological order. That worked well when it was strictly number ranges, as "1000-1099" is > "900-999". However, the BCE tags would interfere and get interweaved, so I wrote a custom sort which would accommodate that as well as the "Pre-" tag to show up before all AD dates and keep the tags in chronological order.